docs/installation.md

```diff
@@ -1,10 +1,10 @@
 # Pyserini: Detailed Installation Guide
 
-Pyserini is built on Python 3.10 (other versions might work, but YMMV).
+Pyserini is built on Python 3.10 (other versions might not work).
 At a high level, we try to keep our [`requirements.txt`](../requirements.txt) up to date.
 Pyserini has a number of important dependencies:
 
-+ For sparse retrieval, Pyserini depends on [Anserini](http://anserini.io/), which is built on Lucene.
++ For sparse retrieval, Pyserini depends on [Anserini](http://anserini.io/), which is built on Lucene. Make sure that Anserini is up to date, otherwise version compatibility may be an issue.
```

Comment:
if you pip install, you don't need this... this only applies if you want a dev installation - further down.